client/src/components/Main/index.js

```diff
@@ -2,13 +2,16 @@ import React, { useEffect, useState } from "react";
 import { useHistory } from "react-router-dom";
 import ClientService from "../../services";
 import AppBar from "../Common/AppBar";
+import { eventPink, eventYellow } from "../../resources/style-constants";
```

Comment:
You shouldn't need to touch this file, so you should revert your changes in this file.